src/stories/Forms/Button/index.tsx

```diff
@@ -1,29 +1,65 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Spinner } from '../../Foundation/Spinner';
+import { BlueTonesEnum, DarkGrayTonesEnum } from '../../../shared/colors';
 
-const ButtonElement = styled.button<ButtonProps>`
+const ButtonStateColors = {
+  lavender: {
+    default: BlueTonesEnum.lavender,
+    hover: BlueTonesEnum.salviaBlue,
+    active: BlueTonesEnum.royalBlue,
+  },
+  black: {
+    default: 'black',
+    hover: DarkGrayTonesEnum.grey25,
+    active: DarkGrayTonesEnum.grey15,
+  },
+};
+
+const ButtonElement = styled.button<{
+  height: ButtonSizes;
+  secondary: boolean;
+  color: 'black' | 'lavender';
+  disabled: boolean;
+}>`
+  --button-color: ${(p) => `var(--color-${p.color})`};
   display: inline-flex;
   position: relative;
   justify-content: center;
   align-items: center;
-  height: 38px;
+  height: ${(p) => `${p.height}px`};
   padding: 0 var(--spacing-24);
-  border: solid 2px ${(p) => `var(--color-${p.color})`};
+  border: solid 2px var(--button-color);
   border-radius: 30px;
-  color: ${(p) =>
-    p.secondary ? `var(--color-${p.color})` : 'var(--color-white)'};
-  font-weight: bold;
+  color: ${(p) => (p.secondary ? 'var(--button-color)' : 'var(--color-white)')};
   font-family: var(--font-family);
   background: ${(p) =>
```

Comment:
Secondary should be transparent and not white I think, just like the website.

Reply:
thanks!